src/corelibs/U2View/src/ov_msa/General/MSAGeneralTab.cpp

```diff
@@ -57,65 +58,55 @@ MSAGeneralTab::MSAGeneralTab(MSAEditor *_msa)
     initializeParameters();
     connectSignals();
 
-    copyButton->setEnabled(msa->getUI()->getCopyFormattedSelectionAction()->isEnabled());
-
     U2WidgetStateStorage::restoreWidgetState(savableTab);
 
-#ifdef Q_OS_MAC
```

Comment:
Why did you remove tooltips?

Reply:
In line 108 I copy a  tooltip from the original action